l10n_it_central_journal/views/report_account_central_journal.xml

```diff
@@ -1,164 +1,272 @@
 <?xml version="1.0" encoding="utf-8"?>
 <openerp>
-<data>
-
-<template id="report_giornale">
-    <t t-call="report.html_container">
-        <t t-set="title" t-value="'Stampa libro giornale'"/>
-        <t t-set="l10n_it_count_fiscal_year" t-value="year_footer"/>
-        <t t-call="l10n_it_account.internal_layout">
-            <div class="page">
-                <t t-set="print_details" t-value="1"/>
-                <table style="width:100%; font-size: small;" cellspacing="0">
-                        <style>
-                            table {border: 0px;}
-                            thead {background: #F5F5F5;}
-                            .left_without_line {
-                                text-align:left; vertical-align:text-top; padding:5px;
-                            }
-                            .right_without_line {
-                                text-align:right; vertical-align:text-top; padding:5px;
+    <data>
+        <template id="minimal_layout" inherit_id="report.minimal_layout">
+            <xpath expr="//t[@t-if='subst is True']" position="replace">
+                <t t-if="subst is True">
+                    <script>
```

Comment:
La modifica che fai qui vuol dire che il metodo inserito in `l10n_it_account` non funziona?